flight/targets/sprf3e/fw/pios_board.c

```diff
@@ -0,0 +1,479 @@
+/**
```

Comment:
P.S. I know some of the existing stuff is a little bit of a mess so I understand that's where it came from, but would be nice to do it right for new stuff. 😉